VideoToImage/videotoimage.py

```diff
@@ -0,0 +1,41 @@
+import botocore
+import subprocess as sp
+import os
+import boto3
+import datetime
+
+#No. of frames per second
+FRAMES = int(os.environ.get('FRAMESPERSEC'))
+
+def convertVideoToImage(srcKey, srcBucket):
+    s3 = boto3.resource('s3')
+    localFilename = '/tmp/{}'.format(os.path.basename(srcKey))
+    #Download video to local
+    try:
+        s3.Bucket(srcBucket).download_file(srcKey, localFilename)
+    except botocore.exceptions.ClientError as e:
+        if e.response['Error']['Code'] == "404":
+            print("The object does not exist.")
+        else:
+            raise
+    #Chop video into images
+    cmd='ffmpeg -i '+str(localFilename)+' -r '+str(FRAMES)+' img_%04d.jpg'
+    sp.call(cmd,shell=True)
+
+    #Upload file to s3
+    for imgFile in os.listdir("./"):
+        if imgFile.find("img") != -1:
+            with open(imgFile, "rb") as imageFile:
+                f = imageFile.read()
+                imgByteArr = bytearray(f)
+                destination = 'video-image/'+str(datetime.datetime.now())+str(imgFile)
```

Comment:
the path should be more dynamic and come from the `src_key`. whatever the path that the video came from, we should upload the images to the same folder or at least closely resemble the same file structure that the video came from. That way we can tell the program where to save it, by specifying the path from the video's path.